instruqt-tracks/multi-cloud-service-networking-with-consul/config.yml

```diff
@@ -4,7 +4,7 @@
 version: "3"
 containers:
 - name: cloud-client
-  image: gcr.io/instruqt-hashicorp/consul-cloud-client:v1.10.2
+  image: gcr.io/instruqt-hashicorp/consul-cloud-client-no-license:v1.10.2
```

Comment:
Don't use an embedded license version - we have to do this because if we use the "licensed" version of the container, the container `ENV` settings will override what Instruqt wants to set as environment variables for the secrets (We're already using `HC_CONSUL_LICENSE` and `HC_VAULT_LICENSE` as the secret names elsewhere, which Instruqt tries to set as environment variables; if those environment variables are set in the container definition we never see the Instruqt versions and thus can't set the license properly with the values from the secrets)